<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Campaingn Monitoring</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Ressources.qrc">
    <normaloff>:/Icons/Icons/networking-1.svg</normaloff>:/Icons/Icons/networking-1.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QTabWidget" name="LateraltabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="plotsbytype">
         <attribute name="title">
          <string>Plots by type</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTreeView" name="bytype"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="plotsbyinstruments">
         <attribute name="title">
          <string>Plots by instruments</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTreeView" name="byinstrument"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="RunLayouts">
         <attribute name="title">
          <string>RunLayouts</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTreeView" name="trunlayouts"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_6" rowminimumheight="0,0,1,0,0">
        <item row="2" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Choose a time period and send it to all the monitoring windows &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;with the button &lt;/span&gt;&lt;img src=&quot;:/Icons/Icons/sync.svg&quot; width=&quot;30&quot;/&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt; checked.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>By run/stages</string>
                </property>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <spacer name="horizontalSpacer_3">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QLabel" name="startrun">
                       <property name="text">
                        <string>start run</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_4">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <spacer name="horizontalSpacer">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QPushButton" name="Rtimeselect">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="toolTip">
                        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Display the campaign runtable and one can linked the time with the start/end of runs and stages.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                       </property>
                       <property name="text">
                        <string>Select time from Runs</string>
                       </property>
                       <property name="icon">
                        <iconset resource="../Ressources.qrc">
                         <normaloff>:/Icons/Icons/time5.svg</normaloff>:/Icons/Icons/time5.svg</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>16</width>
                         <height>16</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_2">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <item>
                      <spacer name="horizontalSpacer_5">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QLabel" name="endrun">
                       <property name="text">
                        <string>end run</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_6">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>Setting the time interval manually</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="0" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;from&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QDateEdit" name="IdateEdit">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="displayFormat">
                        <string>yyyy/MM/dd</string>
                       </property>
                       <property name="calendarPopup">
                        <bool>true</bool>
                       </property>
                       <property name="timeSpec">
                        <enum>Qt::UTC</enum>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QTimeEdit" name="ItimeEdit">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="displayFormat">
                        <string>hh:mm:ss</string>
                       </property>
                       <property name="timeSpec">
                        <enum>Qt::UTC</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;to&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <widget class="QDateEdit" name="LdateEdit">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="displayFormat">
                        <string>yyyy/MM/dd</string>
                       </property>
                       <property name="calendarPopup">
                        <bool>true</bool>
                       </property>
                       <property name="timeSpec">
                        <enum>Qt::UTC</enum>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QTimeEdit" name="LtimeEdit">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="displayFormat">
                        <string>hh:mm:ss</string>
                       </property>
                       <property name="timeSpec">
                        <enum>Qt::UTC</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="TimeInterval">
                  <property name="text">
                   <string>Time Interval</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <spacer name="horizontalSpacer_7">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="sendtime">
                    <property name="text">
                     <string>Send Time</string>
                    </property>
                    <property name="icon">
                     <iconset resource="../Ressources.qrc">
                      <normaloff>:/Icons/Icons/doubleforward.svg</normaloff>:/Icons/Icons/doubleforward.svg</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="Last2h">
                  <property name="text">
                   <string>Last 2h</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../Ressources.qrc">
                    <normaloff>:/Icons/Icons/arrow48.svg</normaloff>:/Icons/Icons/arrow48.svg</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="Last4h">
                  <property name="text">
                   <string>Last 4h</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../Ressources.qrc">
                    <normaloff>:/Icons/Icons/left75.svg</normaloff>:/Icons/Icons/left75.svg</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="last8h">
                  <property name="text">
                   <string>Last 8h</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../Ressources.qrc">
                    <normaloff>:/Icons/Icons/doubleback.svg</normaloff>:/Icons/Icons/doubleback.svg</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QPushButton" name="unsync">
                    <property name="text">
                     <string>Unsyncall</string>
                    </property>
                    <property name="icon">
                     <iconset resource="../Ressources.qrc">
                      <normaloff>:/Icons/Icons/unsync.svg</normaloff>:/Icons/Icons/unsync.svg</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_8">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pclose">
                    <property name="text">
                     <string>Quit program</string>
                    </property>
                    <property name="icon">
                     <iconset resource="../Ressources.qrc">
                      <normaloff>:/Icons/Icons/cancel4.svg</normaloff>:/Icons/Icons/cancel4.svg</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="SetRunLayouts">
              <property name="text">
               <string>setRunLayouts</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>original plot interval lenght       </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="horas">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>hours</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="minutos">
              <property name="text">
               <string>20</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>minutes(0 to 59)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1,0,2,5">
          <item>
           <widget class="QPushButton" name="refresh">
            <property name="toolTip">
             <string>Refresh the 3 lists</string>
            </property>
            <property name="text">
             <string>Refresh</string>
            </property>
            <property name="icon">
             <iconset resource="../Ressources.qrc">
              <normaloff>:/Icons/Icons/refresh19.svg</normaloff>:/Icons/Icons/refresh19.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="killall">
            <property name="text">
             <string>Kill</string>
            </property>
            <property name="icon">
             <iconset resource="../Ressources.qrc">
              <normaloff>:/Icons/Icons/Skull_and_crossbones_vector.svg</normaloff>:/Icons/Icons/Skull_and_crossbones_vector.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Monitoring Slave path&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="slavepath"/>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Error output.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
  </widget>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources.qrc">
     <normaloff>:/Icons/Icons/database7.svg</normaloff>:/Icons/Icons/database7.svg</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to database</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Ressources.qrc"/>
  <include location="../Ressources.qrc"/>
 </resources>
 <connections/>
</ui>
